features/step_definitions/lolcommits_steps.rb

```diff
@@ -73,7 +73,7 @@
 end
 
 Then /^I should be (prompted for|presented) "(.*?)"$/ do |_, prompt|
-  assert @stdout.read.to_s.include?(prompt)
+  expect(@stdout.read.to_s).to include(prompt)
```

Comment:
Shouldn't we stick with the assert syntax in this file? Given we're using that style in the refactored plugin tests in this PR?  Personally I prefer the assert MiniTest style over the 'expect to' grammar but this is just mho :smile: